src/config/dependencies.py

```
@@ -101,5 +103,5 @@ def get_s3_storage_client(
         endpoint_url=settings.S3_STORAGE_ENDPOINT,
         access_key=settings.S3_STORAGE_ACCESS_KEY,
         secret_key=settings.S3_STORAGE_SECRET_KEY,
-        bucket_name=settings.S3_BUCKET_NAME
+        bucket_name=settings.S3_BUCKET_NAME,
```

When creating the S3StorageClient, confirm that `settings.S3_BUCKET_NAME` and `settings.S3_STORAGE_ENDPOINT` are present and valid in all environments. If your S3 client requires additional params (region, secure flag, endpoint format) make sure the settings expose them. Also consider validating these settings at startup and raising a helpful error if the bucket/endpoint are missing so tests and e2e runs fail fast with clear diagnostics.